src/tools/BrushTool.js

```diff
@@ -25,10 +20,10 @@ export default class BrushTool extends BaseBrushTool {
   }
 
   /**
-  * Called by the event dispatcher to render the image.
-  *
-  * @param {Object} evt - The event.
-  */
+   * Called by the event dispatcher to render the image.
```

Comment:
Ahh good spot, I've only just started auto-generating these.

Reply:
Our generated `api` docs are pretty sad for `cornerstone tools`, haha. I think we'll need to make a strong effort to add a lot of the doc tags recommended by `documentation.js`